website/docs/r/api_gateway_account.html.markdown

```diff
@@ -10,8 +10,6 @@ description: |-
 
 Provides a settings of an API Gateway Account. Settings is applied region-wide per `provider` block.
 
--> **Note:** As there is no API method for deleting account settings or resetting it to defaults, destroying this resource will keep your account settings intact
```

Comment:
In case some configuration out there may be relying on the no-op destroy behavior, should we consider adding a `skip_destroy` argument to optionally get it back?